src/layout/Title.tsx

```diff
@@ -14,6 +14,25 @@ export default class Title extends React.Component<Props> {
         this.iconUrl = "";
     }
 
+    public componentDidMount() {
+        const container = document.getElementsByClassName("title-game")[0] as HTMLElement;
```

Comment:
Uhm, why are we circumventing React here?

Reply:
I wasn't sure how to get the width of a rendered element without putting code to execute immediately after render. If there is a way and you know it I'm all for it.

Reply:
I feel like this could break if there's multiple title components involved though. I'm pretty sure React has something that gives you the proper element directly.

Reply:
Other than that, I guess I'm fine with it, if this is really necessary.

Reply:
Well, could store it in the state that would allow us to access it directly, but with how the state onliest reacts with lsc I want sure if it was the best idea

Reply:
Also, implemented this because if there is an image and the text is sent to center, if the text is too long it will go under the icon, this circumvents this by checking if it's long at render so and then sets it to size to the right of the image.